.github/workflows/add-release-reviewers.yml

```
@@ -9,29 +9,42 @@ permissions: {}
 
 jobs:
   add-reviewers:
-    if: ${{ github.event.label.name == 'release' }}
     # Tests that label is added on the PR
+    if: ${{ github.event.label.name == 'release' }}
     runs-on: ubuntu-latest
     permissions:
-      contents: read
+      contents: write
```

Not 100% sure, but given [this description](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#overview:~:text=GitHub%20Apps.%22-,contents,-Work%20with%20the) I would assume that it should be enough to have `read` here. 🤔 
```suggestion
      contents: read
```